ledger/byzcoin/transaction.go

```diff
@@ -0,0 +1,146 @@
+package byzcoin
+
+import (
+	"crypto/sha256"
+	fmt "fmt"
+	"hash"
+
+	proto "github.com/golang/protobuf/proto"
+	"go.dedis.ch/fabric/crypto"
+	"go.dedis.ch/fabric/ledger"
+	"golang.org/x/xerrors"
+)
+
+// Digest is the type of a transaction ID.
```

Comment:
Yes, I'm ending up using it in every module but I'm not sure it's a good idea to define it again and again. I guess the question how much we want to define an ID. We can define the length and let the hash algorithm as a choice or keep using a slice of bytes. It would be nice to have a chat about that at some point. For the moment I will keep it that way.